Description: Fix a manpage issue.
 * When manpages are generated from these files, 
   this stuff with the dots is interpreted as a
   (missing) macro and not displayed.
Author: Tobias Hansen <tobias.han@gmx.de>

--- a/docs/src/refman/font.txt
+++ b/docs/src/refman/font.txt
@@ -222,13 +222,13 @@
 The bitmap format is as in the following example, which contains three glyphs
 for 1, 2 and 3.
 
-    .............
-    . 1 .222.333.
-    . 1 .  2.  3.
-    . 1 .222.333.
-    . 1 .2  .  3.
-    . 1 .222.333.
-    .............
+    "............."
+    ". 1 .222.333."
+    ". 1 .  2.  3."
+    ". 1 .222.333."
+    ". 1 .2  .  3."
+    ". 1 .222.333."
+    "............."
 
 In the above illustration, the dot is for pixels having the background color. It
 is determined by the color of the top left pixel in the bitmap. There should be
